### PR TITLE
Fix Earlier Date Range Error

### PR DESCRIPTION
**Pull request by joeosburn-desiringgod**
@benhutton This includes some rubocop updates as well.

### CODE REVIEW

File: app/services/wakes/pageview_count_updater_service.rb [hunk 1]
```diff
@@ -8,14 +8,14 @@ def initialize(location)
 
   def update_pageview_count
     if pageviews_since_last_update.positive?
-      update_location_pageview_count && update_resource_aggregate_count && update_wakeable_aggregate_count
+      (update_location_pageview_count && update_resource_aggregate_count && update_wakeable_aggregate_count).tap do
```
Comment (benhutton) [inline app/services/wakes/pageview_count_updater_service.rb]: @joeosburn-desiringgod why `#tap`?

File: app/services/wakes/pageview_count_updater_service.rb [hunk 2]
```diff
@@ -8,14 +8,14 @@ def initialize(location)
...
+        # Yes, this will often result in two update operations
+        # Right now, code expressiveness and comprehension seem more important than performance
+        location.update(:pageview_count_checked_at => Time.zone.now)
```
Comment (benhutton) [inline app/services/wakes/pageview_count_updater_service.rb]: @joeosburn-desiringgod why not put this in `#update_location_pageview_count`?

Comment (benhutton) [inline app/services/wakes/pageview_count_updater_service.rb]: also, why do you think this timestamp (as opposed to `pageview_count_updated_through`) is where the problem is? Have you figured/reasoned that out?

File: app/services/wakes/pageview_count_updater_service.rb (hunk 1 above)
Comment (joeosburn-desiringgod) [inline app/services/wakes/pageview_count_updater_service.rb]: @benhutton This ensures the result of all three method calls gets returned, which is the intended behavior of this method.

**Comment by joeosburn-desiringgod** [comment]
Closing based on the fact that this does not address the problem that we thought/hoped it would.

**Pull request closed without merging**